gpt_index/readers/string_iterable.py

```diff
@@ -0,0 +1,32 @@
+"""Simple reader that turns an iterable of strings into a list of Documents."""
+from typing import Any, List
+
+from gpt_index.readers.base import BaseReader
+from gpt_index.readers.schema.base import Document
+
+
+class StringIterableReader(BaseReader):
+    """String Iterable Reader.
+
+    Gets a list of documents, given an iterable (e.g. list) of strings.
+
+    Example:
+        .. code-block:: python
+            from gpt_index import StringIterableReader, GPTTreeIndex
+
+            documents = StringIterableReader().load_data(
+                texts=["I went to the store", "I bought an apple"]).load_data()
```

Comment:
nit: do you need two load_data calls?

Reply:
thanks for finding that! https://github.com/jerryjliu/gpt_index/pull/173/commits/1a9ba224ef8a1c68d4af9d20c027eaed90fcc307